src/components/screens/timebank-screen.tsx

```diff
@@ -30,51 +30,83 @@ const TimebankScreen = () => {
   const [personTotalTime, setPersonTotalTime] = useAtom(timebankScreenPersonTotalTimeAtom);
   const [personDailyEntry, setPersonDailyEntry] = useAtom(personDailyEntryAtom);
   const [dailyEntries, setDailyEntries] = useAtom(dailyEntriesAtom);
+  const loggedInPerson = persons.find(
+    (person: Person) => person.id === config.person.forecastUserIdOverride || person.keycloakId === userProfile?.id
+  );
+  const [selectedEmployeeId, setSelectedEmployeeId] = useState(loggedInPerson?.id);
+  const selectedPerson = persons.find((person) => person.id === selectedEmployeeId);
 
   useEffect(() => {
-    getPersonTotalTime();
-  }, [persons, timespan]);
+    if (selectedEmployeeId) {
```

Comment:
While this condition is necessary, it also seems to be the cause (or at least related to) the issue of the data not being loaded when the timebank screen is refreshed. Please investigate so this refresh issue can be resolved, let me know how it goes.

Reply:
I believe adding the following UseEffect should resolve the refresh issue, please test it out:
  useEffect(() => {
    if (loggedInPerson) {
      setSelectedEmployeeId(loggedInPerson.id);
    }
  }, [loggedInPerson])

Reply:
Yes, it really fixes loading. Thx!